docs/specs/servers/blockchain/blockchain-server-api.md

```diff
@@ -276,3 +276,127 @@ The POST request body should be in JSON format with the following structure:
 
 * `400 Bad Request` - some parameters in request body were missed or wrong.
 * `401 Unauthorized` - projectID verification error.
+
+## Conversion
+
+Endpoints to make convert token transactions.
+
+### List of tokens available for conversion
+
+`GET /v1/convert/tokens`
+
+#### Path parameters
+
+* `projectId` - is the project identifier
+* `chainId` - filter by chain. CAIP-2 format. e.g. `eip155:1`
+
+#### Success response body:
+
+* `tokens` - list of objects which define available tokens:
+    * `name` - token name.
+    * `symbol` - token symbol.
+    * `address` - contract address of the token in CAIP-10 format.
+    * `decimals` - decimals for amount supported by a given token.
+    * `logoUri` - URL of the token icon.
+    * `eip2612` - (Optional for ERC-20 tokens) value is `true` if the token supports [eip-2612](https://eips.ethereum.org/EIPS/eip-2612)
+
+#### Response error codes:
+
+* `400 Bad Request` - some parameters in request body were missed or wrong.
+* `401 Unauthorized` - project ID verification error.
+
+### Requesting conversion quotes
+
+`GET /v1/convert/quotes`
+
+#### Path parameters
+
+* `projectId` - is the project identifier.
+* `amount` - amount of tokens to be converted related to the decimals of the token. Numeric value up to 18 decimals.
+* `userAddress` - caller address.
```

Comment:
Unclear what this is for?

Reply:
This is a user address. It is needed for the approve and convert transactions builder in one inch and socket (multichain) as well. I don't know how to describe it better, `userAddress` should be self describing 🤷‍♂️

Reply:
Is it the same or different from `source`/`from`?